docs/build-process/worker-build-process.md

```diff
@@ -0,0 +1,90 @@
+
+To download the schema standard library, run:
+```
+spatial package get --force --unzip schema standard_library <my-sdk-version> <my-download-directory>
+```
+
+### Download the schema compiler
+The schema compiler is a tool for generating SpatialOS components API code from schema. 
+
+To download the schema compiler, run:
+```
+spatial package get --force --unzip tools schema_compiler-x86_64-<my-target-platform> <my-sdk-version> <my-download-directory>
+```
+`<my-target-platform>` denotes the operating system of the machine that will run the schema compiler, and must be one of `win32`, `macos`, or `linux`.
+
+**Tip:** You will also use the schema compiler when you come to generate a schema descriptor. For more information, see [Building a schema descriptor in the FPL](schema-descriptor-build-process.md).
+
+## 2. Generate code
+In this build step, you use the schema compiler to generate API code for interacting with SpatialOS components from schema. You can regard the generated code as library code that your worker calls as part of its simulation logic.
+
+For more details on how to use the schema compiler and a full list of supported operations, see its [documentation](https://docs.improbable.io/reference/latest/shared/schema/introduction#using-the-schema-compiler-directly).
+
+Below is an example of how to generate component API code for C#:
+```
+schema_compiler  --schema_path=<my-schema-dir>  --schema_path=<schema-std-lib-dir>  --csharp_out=<generated-output-dir> --load_all_schema_on_schema_path   <my-schema-dir>/*.schema <schema-std-lib-dir>/*.schema
+```
+**Note:** You need to perform this build step whenever you change your project’s schema as you iterate on your game, to make sure the generated code for your worker is up-to-date.
+
+**Tip:** You may want to add the generated code to your `.gitignore` file or equivalent to avoid checking it into source control.
+
+## 3. Build the worker executable
+In this final build step, you will use a build tool or process of your choice to compile your worker source into an executable binary. Make sure to reference both the worker SDK libraries and the generated API code as build dependencies of your worker.
+
+The build process is fully customisable to the specific requirements of your worker. You might make use of a widely known build tool like CMake or MSBuild, or rely on a build system specific to your game engine.
+
+**Migration advice:** You can easily determine the build process of your existing worker by inspecting the `build` section of its [build configuration file](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build) referenced in its [worker configuration file](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-configuration). In the CPL, the build configuration file specifies which command and arguments the spatial CLI should run under the hood when building a worker. 
```

Comment:
```suggestion
**Migration advice:** You can easily determine the build process of your existing worker by inspecting the [`build` configuration section](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build) of its [worker configuration file](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-configuration). In the CPL, the build configuration section specifies which command and arguments the spatial CLI should run under the hood when building a worker. 
```

Reply:
This makes it sound like the build section is part of the worker config. THe build section is part of the worker build config.